LocalSettings.php

```diff
@@ -4269,6 +4269,10 @@
 		'wmgUseUploadWizard' => [
 			'campaignExpensiveStatsEnabled' => false,
 		],
+		'+gpcommonswiki' => [
+			'flickrApiUrl' => 'https://api.flickr.com/services/rest/?',
```

Comment:
What's the reasoning for this? I think this might be the default but not 100% sure.

Reply:
> What's the reasoning for this? I think this might be the default but not 100% sure.

Tbh, I couldn't even find a single default for the extension on its .json.... So I can't really tell, but I tried using the Flickr stuff just now on UploadWizard and it responded fine, although I didn't try uploading using it
....

Reply:
The defaults are set a different way for this extension.

Reply:
> The defaults are set a different way for this extension.

Yeah,ni observed that.

Reply:
flickrApiUrl can be removed from here I think.